src/Microsoft.NetCore.Analyzers/Core/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1074,4 +1074,10 @@
   <data name="JsonNetMaybeInsecureSerializerTitle" xml:space="preserve">
     <value>Ensure that JsonSerializer has a secure configuration when deserializing</value>
   </data>
+  <data name="DoNotOverloadSerializationBinderWithoutThrowingAnExceptionTitle" xml:space="preserve">
+    <value>Do not overload SerializationBinder without throwing an exception for unexpected types.</value>
```

Comment:
>. [](start = 97, length = 1)

No punctuation in titles

Also, did files under src\Microsoft.NetCore.Analyzers\Core\xlf change?  Should push those as well.